homeassistant/components/netgear_lte.py

```diff
@@ -54,7 +56,13 @@ class ModemData:
             self.serial_number = information.serial_number
             self.unread_count = sum(1 for x in information.sms if x.unread)
             self.usage = information.usage
+            if not self.connected:
+                _LOGGER.warning("Connected to %s", self.host)
```

Comment:
Maybe info level?

Reply:
I think it should match the warnings for lost connections so it is clear that the problem is resolved even if the log file filters out info level logs.

(Note that it does not print during normal startup, only after connection failures.)